src/backend.github.js

```diff
@@ -461,7 +459,7 @@ var _ = Mavo.Backend.register($.Class({
 
 				history.pushState({}, "", `${location.pathname}?${params}`);
 				window.location.replace(`${location.pathname}?${params}`); 
```

Comment:
Same about `window.location` here, you don’t need `window.`. In fact, you never need `window.`.